doc/index.rst

```diff
@@ -1,10 +1,44 @@
-The Plutus Platform
+Plutus Tools SDK User Guide
 ===============================
 
-The Plutus Platform is an application development platform for the Cardano blockchain.
-To read more about the Platform, see :ref:`what_is_the_plutus_platform`.
-To get started using the Platform see :ref:`plutus_getting_started`.
+Intended Audience
+-------------------------------
 
+External Developers
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This user guide is intended for external developers who are authoring distributed 
+applications ("DApps") that implement smart contracts on the Cardano blockchain. 
+
+.. note::
+   If you are a developer who wants to contribute to the Plutus Tools SDK project, 
+   please refer to documentation residing in the `Plutus Tools SDK repository <https://github.com/
+   input-output-hk/plutus-apps>`_.
+
+Plutus Tools SDK
+===============================
+
+The Plutus Tools SDK is a collection of off-chain infrastructure resources built 
+for external developers. The `Plutus Tools SDK repository <https://github.com/
+input-output-hk/plutus-apps>`_ supports the underlying resources that developers 
+need who are writing full applications using Plutus in Haskell, including off-chain 
+code. The term “off-chain code” refers to the part of a contract application’s code 
+which runs off the chain. 
+
+The `Plutus Tools SDK repository <https://github.com/input-output-hk/plutus-apps>`_ 
+contains packages such as: 
+
+* the `Plutus Application Backend (PAB) <https://github.com/input-output-hk/plutus-apps/tree/main/plutus-pab>`_, 
+  an off-chain application for managing the state of Plutus contract instances;
+
+* the `chain-index <https://github.com/input-output-hk/plutus-apps/tree/main/plutus-chain-index>`_, 
+  a lightweight, customizable chain follower application and library for DApp 
+  developers who need to index and query the Cardano blockchain; 
+
+* a variety of other Plutus packages. 
+
+To get started using the Plutus Tools SDK, see the `Plutus starter template repository 
```

Comment:
I suggest: "See the `Plutus starter template repository <https://github.com/input-output-hk/plutus-starter>`_ for a simple starter project using the Plutus Tools SDK.

Reply:
Suggestion incorporated.